app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,12 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/main_hero_list"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layoutManager="androidx.recyclerview.widget.GridLayoutManager"
+        tools:listitem="@layout/hero_item"
+        tools:spanCount="5" />
```

Comment:
El numero de columnas es bueno que se definan desde el codigo con un GridManagerLayout.
Este tipo de definicion de valores por defecto podrian no funcionar como uno quiere.

Reply:
That change is already done on the PR #14 :smiley: